packages/openwrt/etc/init.d/edge

```diff
@@ -3,7 +3,7 @@ START=90
 STOP=10
 
 USE_PROCD=1
-PROG=/usr/bin/edge
+PROG=/usr/bin/n2n-edge
```

Comment:
Changing the name of the binary breaks existing user's expectations.  It also breaks compatibility between this and the non openwrt versions

Reply:
Okay I'll revert this change.